---
layout: default
title: Publications
---

**Table of Contents:**

* TOC
{:toc}

> [Representative Publications](./pub.md)
>
> [代表性文章中文解读](../news/#news)

---



## 2025

* Hongli Ma, Letian Gao, Yunfan Jin, Jianwei Ma, Yilan Bai, Xiaofan Liu, Pengfei Bao, Ke Liu, Zhenjiang Zech Xu\* and **Zhi John Lu**\*  (2025)  **RNA–ligand interaction scoring via data perturbation and augmentation modeling**. *Nature Computational Science* 10.1038/s43588-025-00820-x (\*co-corresponding authors) 

   [View Detail](https://www.nature.com/articles/s43588-025-00820-x)  \|  [Free Full-text](https://rdcu.be/etaiM)  (view-only)


* Pengfei Bao<sup>1</sup>, Taiwei Wang<sup>1</sup>, Xiaofan Liu<sup>1</sup>, Shaozhen Xing, Hanjin Ruan, Hongli Ma, Yuhuan Tao, Qing Zhan, Efres Belmonte-Reche, Lizheng Qin, Zhengxue Han, Minghui Mao\*, Mengtao Li\* & **Zhi John Lu**\* (2025) **Peak analysis of cell-free RNA finds recurrently protected narrow regions with clinical potential.** *Genome Biology* 26, 119 (<sup>1</sup>contributed equally, \*co-corresponding authors) 

   [View Detail](https://doi.org/10.1186/s13059-025-03590-x) 

> —————————————————75————————————————————

## 2024

* Yilan Bai, Haochen Zhong, Taiwei Wang, **Zhi John Lu** (2024) **OligoFormer: An accurate and robust prediction method for siRNA design.** *Bioinformatics* 40 (10), btae577

    [View Detail](https://doi.org/10.1093/bioinformatics/btae577) 

* Hongke Wang<sup>1</sup>, Qing Zhan<sup>1</sup>, Meng Ning<sup>1</sup>, Hongjie Guo<sup>1</sup>, Qian Wang, Jiuliang Zhao, Pengfei Bao, Shaozhen Xing, Shanwen Chen, Shuai Zuo, Xuefeng Xia, Mengtao Li\*, Pengyuan Wang\*, **Zhi John Lu**\* (2024) **Depletion-assisted multiplexed cell-free RNA sequencing reveals distinct human and microbial signatures in plasma versus extracellular vesicles** *Clinical and Translational Medicine* 14:e1760 (<sup>1</sup>contributed equally, \*co-corresponding authors) 

    [View Detail](http://dx.doi.org/10.1002/ctm2.1760) 
    
* Genpeng Li<sup>1</sup> , Wang Hongke<sup>1</sup> , Jinjing Zhong , Yilan Bai , Chen Wenjie , Ke Jiang , Jing Huang , Yuting Shao , Jiaye Liu , Gong Yanping, Zhang Junhui , Sun Ronghao , Tao Wei , Rixiang Gong , Zhu Jingqiang , **Zhi Lu** , Zhihui Li , Jianyong Lei (2024)  **Circulating small extracellular vesicle-based miRNA classifier for follicular thyroid carcinoma: a diagnostic study** *BRITISH JOURNAL OF CANCER* 130(6):925-933 (<sup>1</sup>contributed equally) 

    [View Detail](https://pubmed.ncbi.nlm.nih.gov/38238428/) 
    
* Xiaofan Liu<sup>1</sup>, Yuhuan Tao<sup>1</sup>, Zilin Cai, Pengfei Bao, Hongli Ma, Kexing Li, Mengtao Li\*, Yunping Zhu\*, **Zhi John Lu**\*  (2024) **Pathformer: a biological pathway informed Transformer for disease diagnosis and prognosis using multi-omics data** *Bioinformatics* 40(5), btae316 (<sup>1</sup>contributed equally, \*co-corresponding authors) 

    [View Detail](https://doi.org/10.1093/bioinformatics/btae316) 

* Mingyang Li<sup>1</sup>, Tianxiu Zhou<sup>1</sup>, Mingfei Han<sup>1</sup>, Hongke Wang, Pengfei Bao, Yuhuan Tao, Xiaoqing Chen, Guansheng Wu, Tianyou Liu, Xiaojuan Wang, Qian Lu\*, Yunping Zhu\*, **Zhi John Lu**\* (2024) **cfOmics: a cell-free multi-Omics database for diseases.**  *Nucleic Acids Research* 52(D1):D607-D621 (<sup>1</sup>contributed equally, \*co-corresponding authors) 

    [View Detail](https://doi.org/10.1093/nar/gkad777) 

* 刘晓帆, 鲁志. **复杂疾病中多组学多模态数据的生物信息学研究进展**. *科学通报*, 2024, 69: 1–15
     Xiaofan Liu and  **Zhi Lu** (2024) **Progress of bioinformatics studies for multi-omics and multi-modal data in complex diseases** (in Chinese). *Chinese Science Bulletin*, 69: 1–15, doi: 10.1360/TB-2024-0416
     
    [View Detail](https://doi.org/10.1360/TB-2024-0416) 


> —————————————————69————————————————————

## 2023

* Yuhuan Tao<sup>1</sup>, Shaozhen Xing<sup>1</sup>, Shuai Zuo<sup>1</sup>, Pengfei Bao, Yunfan Jin, Yu Li, Yingchao Wu, Shanwen Chen, Xiaojuan Wang, Yumin Zhu, Ying Feng, Xiaohua Zhang, Xianbo Wang, Qiaoran Xi, Qian Lu\*, Pengyuan Wang\*, **Zhi John Lu**\*  (2023) **Cell-free multi-omics analysis reveals potential biomarkers in gastrointestinal cancer patients’ blood.** *Cell Reports Medicine* 4(11):101281 (<sup>1</sup>contributed equally, \*co-corresponding authors) 

    [View Detail](https://www.cell.com/cell-reports-medicine/fulltext/S2666-3791(23)00475-5) 


* Xing, S.<sup>1</sup>, Zhu, Y.<sup>1</sup>, You, Y., Wang, S., Wang, H., Ning, M., Jin, H., Liu, Z., Zhang, X., Yu, C. \* & **Lu, Z. J.**\* (2023). **Cell-free RNA for the liquid biopsy of gastrointestinal cancer.** *WIREs RNA* 14(5):e1791 (<sup>1</sup>contributed equally, \*co-corresponding authors) 

    [View Detail](https://doi.org/10.1002/wrna.1791)   \|  [PDF](https://cloud.tsinghua.edu.cn/d/07d2b19d6b284ebea5ea/?p=%2FLu%20Lab%20Publications%2F2023&mode=list)


* Chun Ning<sup>1</sup>, Peng Cai<sup>1</sup>, Xiaofan Liu<sup>1</sup>, Guangtao Li<sup>1</sup>, Pengfei Bao, Lu Yan, Meng Ning, Kaichen Tang, Yi Luo, Hua Guo, Lu Chen\*, **Zhi John Lu** \*, Jianhua Yin\* (2023) **A Comprehensive Evaluation of Full-spectrum Cell-free RNAs highlights cell-free RNA fragments for Early-Stage Hepatocellular Carcinoma Detection** *eBioMedicine* 93:104645 (<sup>1</sup>contributed equally, \*co-corresponding authors) 

   [View Detail](https://pubmed.ncbi.nlm.nih.gov/37315449/)   


> Yunfan Jin, Hongli Ma, Zhenjiang Zech Xu\* and **Zhi John Lu**\*.  **BATTER: Accurate Prediction of Rho-dependent and Rho-independent Transcription Terminators in Metagenomes** bioRxiv 10.1101/2023.10.02.560326  (\*co-corresponding authors)    [Preprint](https://doi.org/10.1101/2023.10.02.560326) 


> —————————————————66————————————————————


## 2022

* Shanwen Chen<sup>1</sup>, Yunfan Jin<sup>1</sup>, Siqi Wang<sup>1</sup>, Shaozhen Xing<sup>1</sup>, Yingchao Wu, Yuhuan Tao, Yongchen Ma, Shuai Zuo, Xiaofan Liu, Yichen Hu, Hongyan Chen, Yuandeng Luo, Feng Xia, Chuanming Xie, Jianhua Yin, Xin Wang, Zhihua Liu, Ning Zhang, Zhenjiang Zech Xu\*, **Zhi John Lu**\*, Pengyuan Wang\* (2022) **Cancer Type Classification Using Plasma Cell Free RNAs Derived from Human and Microbes.** *eLife* 11:e75181 (<sup>1</sup>contributed equally, *co-corresponding authors) 

    [View Detail](https://pubmed.ncbi.nlm.nih.gov/35816095/) <img src="./assets/2022_elife.webp" style="zoom:30%;" />


* Haipeng Fu, Tingyu Wang, Xiaohui Kong, Kun Yan, Yang Yang, Jingyi Cao, Yafei Yuan, Nan Wang, Kehkooi Kee, **Zhi John Lu** & Qiaoran Xi (2022) **A Nodal enhanced micropeptide NEMEP regulates glucose uptake during mesendoderm differentiation of embryonic stem cells.** *Nature Communications* 13(1):3984

    [View Detail](https://pubmed.ncbi.nlm.nih.gov/35810171/) 
    
* Liuqing Yang, Kequan Lin, Lin Zhu, Huili Wang, Shuaishuai Teng, Lijun Huang, Shiyi Zhou, Guanbin Zhang, **Zhi John Lu** & Dong Wang (2022) **Long non-coding RNA DARS-AS1 promotes tumor progression by directly suppressing PACT-mediated cellular stress.**  *Communications Biology* 5, 822

    [View Detail](https://doi.org/10.1038/s42003-022-03778-y) 

* Weihao Zhao<sup>1</sup>, Shang Zhang<sup>1</sup>, Yumin Zhu, Xiaochen Xi, Pengfei Bao, Ziyuan Ma, Thomas H Kapral, Shuyuan Chen, Bojan Zagrovic, Yucheng T Yang\*, **Zhi John Lu**\* (2022) **POSTAR3: an updated platform for exploring post-transcriptional regulation coordinated by RNA-binding proteins.** *Nucleic Acids Research* 50(D1):D287-D294 (<sup>1</sup>contributed equally, \*co-corresponding authors)

    [View Detail](https://pubmed.ncbi.nlm.nih.gov/34403477/)  \| [Website](http://POSTAR.ncrnalab.org)

* Thomas H Kapral, Fiona Farnhammer, Weihao Zhao, **Zhi J Lu** and Bojan Zagrovic (2022) **Widespread autogenous mRNA–protein interactions detected by CLIP-seq.** *Nucleic Acids Research*, 50(17):9984-9999

    [View Detail](https://doi.org/10.1093/nar/gkac756) 


> —————————————————61————————————————————

## 2017 - 2021 (Tenured - Phase I)

**2021**

* Yumin Zhu<sup>1</sup>, Siqi Wang<sup>1</sup>, Xiaochen Xi<sup>1</sup>, Minfeng Zhang<sup>1</sup>, Xiaofan Liu, Weina Tang, Peng Cai, Shaozhen Xing, Pengfei Bao, Yunfan Jin, Weihao Zhao, Yinghui Chen, Huanan Zhao, Xiaodong Jia, Shanshan Lu, Yinying Lu, Lei Chen\*, Jianhua Yin\*, **Zhi John Lu**\* (2021) **Integrative analysis of long extracellular RNAs reveals a detection panel of noncoding RNAs for liver cancer.** *Theranostics* 11(1):181-193 (<sup>1</sup>contributed equally, \*co-corresponding authors)

    [View Detail](https://pubmed.ncbi.nlm.nih.gov/33391469/)

* Jing Gong<sup>1</sup>, Kui Xu<sup>1</sup>, Ziyuan Ma, **Zhi John Lu** & Qiangfeng Cliff Zhang (2021) **A deep learning method for recovering missing signals in transcriptome-wide RNA structure profiles from probing experiments.** *Nature Machine Intelligence* 3:995–1006

    [View Detail](https://www.nature.com/articles/s42256-021-00412-0)

* Jia Li, Lei Sun, Xue-Liang Peng, Xiao-Ming Yu, Shao-Jun Qi, **Zhi John Lu**, Jing-Dong J Han, Qin Shen (2021) **Integrative genomic analysis of early neurogenesis reveals a temporal genetic program for differentiation and specification of preplate and Cajal-Retzius neurons**. *PLoS Genetics* 17(3):e1009355
  [View Detail](https://pubmed.ncbi.nlm.nih.gov/33760820/)

* Shanwen Chen, Tao Liu, Dingfang Bu, Jing Zhu, Xin Wang, Yisheng Pan, Yucun Liu, **Zhi John Lu**, Pengyuan Wang (2021）**Methylome profiling identifies TCHH methylation in CfDNA as a noninvasive marker of liver metastasis in colorectal cancer**. *FASEB J.* 35(7):e21720

  [View Detail](https://pubmed.ncbi.nlm.nih.gov/34110642/)


**2020**

* Shuaishuai Teng<sup>1</sup>, Yang Eric Li<sup>1</sup>, Ming Yang, Rui Qi, Yiming Huang, Qianyu Wang, Yanmei Zhang, Shanwen Chen, Shasha Li, Kequan Lin, Yang Cao, Qunsheng Ji, Qingyang Gu, Yujing Cheng, Zai Chang, Wei Guo, Pengyuan Wang, Ivan Garcia-Bassets, **Zhi John Lu**\* and Dong Wang\* (2020) **Tissue-specific transcription reprogramming promotes liver metastasis of colorectal cancer.** *Cell Research* 30(1):34-49 (<sup>1</sup>contributed equally, \*co-corresponding authors)

  [View Detail](https://www.ncbi.nlm.nih.gov/pubmed/31811277)

* Yu-Xiang Chen<sup>1</sup>, Zhi-yu Xu<sup>1</sup>, Xueliang Ge, Suparna Sanyal, **Zhi John Lu**\* and Babak Javid\* (2020) **Selective translation by alternative bacterial ribosomes.** *PNAS* 117(32):19487-19496 (<sup>1</sup>contributed equally, \*co-corresponding authors)

  [View Detail](https://doi.org/10.1073/pnas.2009607117)

* Yu Ma<sup>1</sup>, Shang Zhang<sup>1</sup>, Chao Bi<sup>1</sup>, Chao Mei, Shang-Chuan Jiang, Xiao-Fang Wang,  **Zhi John Lu**\* and Da-Peng Zhang\* (2020) **Arabidopsis exoribonuclease USB1 interacts with the PPR-domain protein SOAR1 to negatively regulate abscisic acid signaling.** *Journal of Experimental Botany* 71(19): 5837-5851 (<sup>1</sup>contributed equally, \*co-corresponding authors)

  [View Detail](https://doi.org/10.1093/jxb/eraa315)
  
* Kequan Lin<sup>1</sup>, Lu Li<sup>1</sup>, Yifei Dai, Huili Wang, Shuaishuai Teng, Xilinqiqige Bao, **Zhi John Lu**\* and Dong Wang\* (2020) **A comprehensive evaluation of connectivity methods for L1000 data** *Briefings in Bioinformatics* 21 (6), 2194-2205 (<sup>1</sup>contributed equally, \*co-corresponding authors)
  
  [View Detail](https://www.ncbi.nlm.nih.gov/pubmed/31774912)


> —————————————————53————————————————————

**2019**

* Rui Xiao<sup>1,</sup>\*, Jia-Yu Chen<sup>1</sup>, Zhengyu Liang<sup>1</sup>, Daji Luo, Geng Chen, **Zhi John Lu**, Yang Chen, Bing Zhou, Hairi Li, Xian Du, Yang Yang, Mingkui San, Xintao Wei, Wen Liu, Eric Lecuyer, Brenton R. Graveley, Gene W. Yeo, Christopher B. Burge, Michael Q. Zhang, Yu Zhou and Xiang-Dong Fu\* (2019) **Pervasive Chromatin-RNA Binding Protein Interactions Enable RNA-Based Regulation of Transcription** *Cell* 178, 107–121 (<sup>1</sup>contributed equally, \*co-corresponding authors)

  [View Detail](https://www.ncbi.nlm.nih.gov/pubmed/31251911)

* Maoguo Luo, Jianbo Bai, Bofeng Liu, Peiqiang Yan, Feifei Zuo, Hongyao Sun, Ye Sun, Xuanhao Xu, Zhihong Song, Yang Yang, Joan Massagué, Xun Lan, **Zhi Lu**, Ye-Guang Chen, Haiteng Deng, Wei Xie, Qiaoran Xi (2019) **H3K18ac Primes Mesendodermal Differentiation upon Nodal Signaling** *Stem Cell Reports* 13(4):642-656

  [View Detail](https://www.ncbi.nlm.nih.gov/pubmed/31564646)

- Chang Tan<sup>1</sup>, Jingyi Cao<sup>1</sup>, Lu Chen<sup>1</sup>, Xiaochen Xi, Siqi Wang, Yumin Zhu, Liuqing Yang, Longteng Ma, Dong Wang, Jianhua Yin\*, Ti Zhang\* and **Zhi John Lu\*** (2019) **Noncoding RNAs serve as diagnosis and prognosis biomarkers for hepatocellular carcinoma** *Clinical Chemistry* 65(7):905-915  (<sup>1</sup>contributed equally, \*co-corresponding authors)

  [View Detail](https://www.ncbi.nlm.nih.gov/pubmed/30996051)

- Yumin Zhu<sup>1</sup>, Gang Xu<sup>1</sup>, Yucheng T Yang<sup>1</sup>, Zhiyu Xu, Xinduo Chen, Binbin Shi, Daoxin Xie, **Zhi John Lu\***, Pengyuan Wang\* (2019) **POSTAR2: deciphering the post-transcriptional regulatory logics** *Nucleic Acids Research* (D1):D203-D211  (<sup>1</sup>contributed equally, \*co-corresponding authors)

  [View Detail](https://www.ncbi.nlm.nih.gov/pubmed/30239819)

- Shasha Li<sup>1</sup>, Shuaishuai Teng<sup>1</sup>, Junquan Xu, Guannan Su, Yu Zhang, Jianqing Zhao, Suwei Zhang, Haiyan Wang, Wenyan Qin, **Zhi John Lu**, Yong Guo, Qianyong Zhu and Dong Wang (2019) **Microarray is an efficient tool for circRNA profiling.** *Briefings in Bioinformatics* 20(4):1420-1433  (<sup>1</sup>contributed equally)

  [View Detail](https://doi.org/10.1093/bib/bby006)

**2018**

- Zhiyu Xu<sup>1</sup>, Long Hu<sup>1</sup>, Binbin Shi, SiSi Geng, Longchen Xu, Dong Wang\* and **Zhi John Lu\*** (2018) **Ribosome elongating footprints denoised by wavelet transform comprehensively characterize dynamic cellular translation events.** *Nucleic Acids Research* 46 (18), e109-e109  (<sup>1</sup>contributed equally, \*co-corresponding authors)

  [View Detail](https://www.ncbi.nlm.nih.gov/pubmed/29945224) [GitHub](https://lulab.github.io/Ribowave)


- \[spotlight\]: Binbin Shi and **Zhi John Lu**\* (2018) **Decoding Plant RNA Structurome.**  *Molecular Plant* 11:3

  [View Detail](https://www.cell.com/molecular-plant/abstract/S1674-2052(18)30099-6)

- J Yuan<sup>1</sup>, J Li<sup>1</sup>, Y Yang, C Tan, Y Zhu, L Hu, Y Qi\*, **ZJ Lu\*** (2018) **Stress‐responsive regulation of long non‐coding RNA polyadenylation in Oryza sativa.** *The Plant Journal* 93:5, 814-827  (<sup>1</sup>contributed equally, \*co-corresponding authors)

  [View Detail](http://onlinelibrary.wiley.com/doi/10.1111/tpj.13804/full)

- J Gong, D Shao, K Xu, Z Lu, **ZJ Lu**, YT Yang, QC Zhang (2018) **RISE: a database of RNA interactome from sequencing experiments.** *Nucleic Acids Research* 46 (D1), D194-D201

  [View Detail](https://academic.oup.com/nar/article/46/D1/D194/4344571)


> —————————————————44————————————————————

**2017**

- Yang Eric Li<sup>1</sup>, Mu Xiao<sup>1</sup>, Binbin Shi<sup>1</sup>, Yu-Cheng T. Yang, Dong Wang, Fei Wang, Marco Marcia and **Zhi John Lu*** (2017) **Identification of high-confidence RNA regulatory elements by combinatorial classification of RNA-protein binding sites.** *Genome Biology* 18:169  (<sup>1</sup>contributed equally)

  [View Detail](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1298-8) [ePDF](http://rdcu.be/vHHS) [Web Site](http://rnatarget.ncrnalab.org/RBPgroup) [GitHub](https://github.com/lulab/RBPgroup)

- Yang Yang<sup>1</sup>, Lei Chen<sup>1</sup>, Jin Gu<sup>1</sup>, Hanshuo Zhang<sup>1</sup>, Jiapei Yuan, Qiuyu Lian, Guishuai Lv, Siqi Wang, Yang Wu, Yu-Cheng T. Yang, Dongfang Wang, Yang Liu, Jing Tang, Guijuan Luo, Yang Li, Long Hu, Xinbao Sun, Dong Wang, Mingzhou Guo, Qiaoran Xi, Jianzhong Xi, Hongyang Wang\*, Michael Q. Zhang\* and **Zhi John Lu\*** (2017) **Recurrently deregulated lncRNAs in hepatocellular carcinoma.** *Nature Communications* 8, 14421  (<sup>1</sup>contributed equally, \*co-corresponding authors)

  [View Details](http://www.nature.com/articles/ncomms14421) [ePDF](http://rdcu.be/pdr7)

- Xiaochen Xi<sup>1</sup>, Tianxiao Li<sup>1</sup>, Yiming Huang, Jiahui Sun, Yumin Zhu, Yang Yang\* and **Zhi John Lu\*** (2017) **RNA Biomarkers: Frontier of Precision Medicine for Cancer.** *Non-Coding RNA*3, 9  (<sup>1</sup>contributed equally, \*co-corresponding authors)

  [View Details](http://www.mdpi.com/2311-553X/3/1/9/)

- Yang Yang<sup>1</sup>, Yu-Cheng T. Yang<sup>1</sup>, Jiapei Yuan, **Zhi John Lu\*** and Jingyi Jessica Li\* (2017) **Large-scale mapping of mammalian transcriptomes identifies conserved genes associated with different cell differentiation states.** *Nucleic Acids Research* 45 (4): 1657-1672  (<sup>1</sup>contributed equally, \*co-corresponding authors)

  [View Details](https://www.ncbi.nlm.nih.gov/pubmed/27980097)

- Zhonglin Tang<sup>1,</sup>\*, Yang Wu<sup>1</sup>, Yalan Yang, Yu-Cheng T. Yang, Zishuai Wang, Jiapei Yuan, Yang Yang, Chaoju Hua, Xinhao Fan, Guanglin Niu, Yubo Zhang, **Zhi John Lu\*** and Kui Li\* (2017) **Comprehensive analysis of long non-coding RNAs highlights their spatio-temporal expression patterns and evolutional conservation in Sus scrofa.** *Scientific Reports* 7:43166 (<sup>1</sup>contributed equally, * co-corresponding authors)

  [View Details](http://www.nature.com/articles/srep43166) [ePDF](http://rdcu.be/pzcF)

- Dan Gao, Yingjie Han, Yang Yang, James G Herman, Enqiang Linghu, Qimin Zhan, François Fuks, **Zhi John Lu**, Mingzhou Guo (2017) **Methylation of TMEM176A is an independent prognostic marker and is involved in human colorectal cancer development.** *Epigenetics*12(7):575-583

  [View Details](https://www.ncbi.nlm.nih.gov/pubmed/28678648)

- Long Hu, Zhiyu Xu, Boqin Hu and **Zhi John Lu\***  (2017)  **COME: a robust coding potential calculation tool for lncRNA identification and characterization based on multiple features. **  *Nucleic Acids Research* 45 (1): e2



  [View Details](https://www.ncbi.nlm.nih.gov/pubmed/27608726) [Web Site](http://rnafinder.ncrnalab.org/COME) [GitHub](https://github.com/lulab/COME)

- Boqin Hu<sup>1</sup>, Yu-Cheng T. Yang<sup>1</sup>, Yiming Huang, Yumin Zhu and **Zhi John Lu\*** (2017) **POSTAR: a platform of post-transcriptional regulation coordinated by RNA binding proteins.** *Nucleic Acids Research* 45 (D1): D104-D114 (<sup>1</sup>contributed equally)

  [View Details](https://www.ncbi.nlm.nih.gov/pubmed/28053162) [Web Site](http://rnatarget.ncrnalab.org/POSTAR)


> —————————————————36————————————————————

## 2012 - 2016 (Tenure Track)

**2016**

- Mengrong Liu<sup>1</sup>, Yu-Cheng T. Yang<sup>1</sup>, Gang Xu, Chang Tan, and **Zhi John Lu*** (2016) **CCG: an integrative resource of cancer protein-coding and long noncoding RNA genes.** *Discovery Medicine* 22 (123): 351-359 (<sup>1</sup>contributed equally)

  [View Details](https://www.ncbi.nlm.nih.gov/pubmed/28147217)

- Yang Wu<sup>1</sup>, Rihao Qu<sup>1</sup>, Yiming Huang<sup>1</sup>, Binbin Shi, Mengrong Liu, Yang Li and **Zhi John Lu\*** (2016) **RNAex: an RNA secondary structure prediction server enhanced by high-throughput structure probing data.** *Nucleic Acids Research* 44 (W1): W294-W301 (<sup>1</sup>contributed equally)

  [View Details](http://nar.oxfordjournals.org/content/44/W1/W294.abstract) [Web Site](http://rnaex.ncrnalab.org/)

- Jiapei Yuan<sup>1</sup>, Ye Zhang<sup>1</sup>, Jinsong Dong, Xinbao Sun, Yuzhe Sun, Boon L. Lim, Dong Liu\* and **Zhi John Lu\*** (2016) **Systematic characterization of novel lncRNAs responding to phosphate starvation in Arabidopsis thaliana.** *BMC Genomics* 17:655 (<sup>1</sup>contributed equally, *: co-corresponding authors)

  [View Details](http://www.ncbi.nlm.nih.gov/pubmed/27538394)

- Baoping Cao<sup>1</sup>, Nanxi Song<sup>1</sup>, Meiying Zhang, Runsheng Chen, Youyong Lu, Runshen Chen\*, **Zhi John Lu\*** and Mingzhou Guo\* (2016) **Systematic study of novel lncRNAs in different gastrointestinal cancer cell lines.** *Discovery Medicine* 21(115):159-171 (<sup>1</sup>contributed equally, *: co-corresponding authors)

  [View Details](http://www.ncbi.nlm.nih.gov/pubmed/27115166)

- \[review\]: Xihao Hu<sup>1</sup>, Yang Wu<sup>1</sup>, **Zhi John Lu** and Kevin Y. Yip. (2016) **Analysis of Sequencing Data for Probing RNA Secondary Structures and Protein-RNA Binding in Studying Post-Transcriptional Regulations.** *Briefings in Bioinformatics* 17(6): 1032-1043 (<sup>1</sup>contributed equally)

  [View Details](https://academic.oup.com/bib/article/17/6/1032/2606441)

- Kejia Wen<sup>1</sup>, Lijuan Yang<sup>1</sup>, Tuanlin Xiong, Chao Di, Danhui Ma, Menghua Wu, Zhaoyu Xue, Xuedi Zhang, Li Long, Weimin Zhang, Jiaying Zhang, Xiaolin Bi, Junbiao Dai, Qiangfeng Zhang, **Zhi John Lu** and Guanjun Gao (2016) **Critical roles of long noncoding RNAs in Drosophila spermatogenesis.** *Genome Research* 26: 1233-1244 (<sup>1</sup>contributed equally)

  [View Details](http://genome.cshlp.org/content/26/9/1233.abstract)

- Sibo Tao<sup>1</sup>, Ye Zhang<sup>1</sup>, Xiaoyue Wang, Xiaofeng Fang, Le Xu, **Zhi John Lu** and Dong Liu (2016) **The THO/TREX complex active in mRNA export suppresses root-associated acid phosphatase activity induced by phosphate starvation.** *Plant Physiology* 171(4):2841-53 (<sup>1</sup>contributed equally)

  [View Details](http://www.ncbi.nlm.nih.gov/pubmed/27329222)

- Sai Luo, J. Yuyang Lu, Lichao Liu, Yafei Yin, Chunyan Chen, Xue Han, Bohou Wu, Ronggang Xu, Wei Liu, Pixi Yan, Wen Shao, **Zhi Lu**, Haitao Li, Jie Na, Fuchou Tang, Jianlong Wang, Yong E. Zhang, Xiaohua Shen (2016) **Divergent lncRNAs Regulate Gene Expression and Lineage Differentiation in Pluripotent Cells.** *Cell Stem Cell* 21(115):159-171

  [View Details](http://www.ncbi.nlm.nih.gov/pubmed/26996597)


**2015**

- Yang Wu<sup>1</sup>, Binbin Shi<sup>1</sup>, Xinqiang Ding, Tong Liu, Xihao Hu, Kevin Y. Yip, Zheng Rong Yang, David H. Mathews and **Zhi John Lu\*** (2015) **Improved prediction of RNA secondary structure by integrating the free energy model with restraints derived from experimental probing data.** *Nucleic Acids Research* 43(15): 7247-59 (<sup>1</sup>contributed equally)

  [View Details](http://www.ncbi.nlm.nih.gov/pubmed/26170232) [GitHub](https://github.com/lulab/RME)

- Long Hu, Chao Di, Mingxuan Kai, Yu-Cheng T. Yang, Yang Li, Yunjiang Qiu, Xihao Hu, Kevin Y. Yip, Michael Q. Zhang and **Zhi John Lu\*** (2015) **A common set of distinct features that characterize noncoding RNAs across multiple species.** *Nucleic Acids Research* 43(1): 104-114

  [View Details](http://www.ncbi.nlm.nih.gov/pubmed/25505163) [Web Site](http://rnafinder.ncrnalab.org/RNAfeature)

- Yu-Cheng T. Yang<sup>1</sup>, Chao Di<sup>1</sup>, Boqin Hu<sup>1</sup>, Meifeng Zhou, Yifang Liu, Nanxi Song, Yang Li, Jumpei Umetsu and **Zhi John Lu\*** (2015) **CLIPdb: a CLIP-seq database for protein-RNA interactions.** *BMC Genomics* 16:51 (<sup>1</sup>contributed equally)

  [View Details](http://www.ncbi.nlm.nih.gov/pubmed/25505163) [Web Site](http://rnatarget.ncrnalab.org/CLIPdb)

- Yinghan Fu, Zhengjiang Zech Xu, **Zhi J. Lu**, Shan Zhao, and David H. Mathews(2015) **Discovery of Novel ncRNA Squences in Multiple Genome Alignments on the Basis of Conserved and Stable Secondary Structures.** *PloS one*10(6):e0130200

  [View Details](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0130200)


> —————————————————24————————————————————

**2014**

- Chao Di<sup>1</sup>, Jiapei Yuan<sup>1</sup>, Yue Wu, Jingrui Li, Huixin Lin, Long Hu, Ting Zhang, Yijun Qi, Mark B. Gerstein, Yan Guo and **Zhi John Lu\*** (2014) **Characterization of Stress-responsive lncRNAs in Arabidopsis thaliana by Integrating Expression, Epigenetic and Structural Features.** *The Plant Journal* 80(5), 848-861 (<sup>1</sup>contributed equally)

  [View Details](http://www.ncbi.nlm.nih.gov/pubmed/25256571)

- Mark B. Gerstein, Joel Rozowsky, Koon-Kiu Yan, Daifeng Wang, Chao Cheng, James B. Brown, Carrie A. Davis, LaDeana Hillier, Cristina Sisu, Jingyi Jessica Li, Baikang Pei, Arif O. Harmanci, Michael O. Duff, Sarah Djebali, Roger P. Alexander,... Chao Di, ... **Zhi Lu**, ... Thomas R. Gingeras and Robert Waterston.(2014) **Comparative analysis of the transcriptome across distant species.** *Nature* 512:445-448

  [View Details](http://www.nature.com/nature/journal/v512/n7515/abs/nature13424.html)

- YC Yang, J Umetsu, **ZJ Lu\*** (2014) **Global signatures of protein binding on structured RNAs in Saccharomyces cerevisiae.** *Science China Life Sciences* 57 (1), 22-35 (Tsinghua Special Issue)

  [View Details](http://www.ncbi.nlm.nih.gov/pubmed/24369346)

- Hu X, Wong TKF, **Lu ZJ**, Chan TF, Lau TCK, Yiu SM and Yip KY. (2014) **Computational Identification of Protein Binding Sites on RNAs using High-Throughput RNA Structure-Probing Data.** *Bioinformatics* 30(8), 1049-1055.

  [View Details](http://bioinformatics.oxfordjournals.org/content/30/8/1049)

- B Ren, H Shen, **ZJ Lu**, H Liu, Y Xu. (2014) **The phzA2-G2 Transcript Exhibits Direct RsmA-Mediated Activation in Pseudomonas aeruginosa M18.** *PloS one* 9 (2), e89653

  [View Details](http://scholar.qsensei.com/content/1zjdv6/)



**2013**

* Freeberg MA, Han T,Moresco JJ, Kong A, Yang YC, **Lu ZJ**, Yates JR, Kim JK. (2013) **Pervasive and dynamic protein binding sites of the mRNA transcriptome in Saccharomyces cerevisiae.** *Genome Biology* 14, R13.

  [View Details](http://www.ncbi.nlm.nih.gov/pubmed/23409723/)

* Kudron M, Niu W, **Lu Z**, Wang G, Gerstein M, Snyder M, Reinke V. (2013) **Tissue-specific direct targets of Caenorhabditis elegans Rb/E2F dictate distinct somatic and germline programs.** *Genome Biology* 14:R5

  [View Details](http://genomebiology.com/2013/14/1/R5)


**2012**

* Tan X, Hu L<sup>2</sup>, Luquette JL, Gao G, Liu YF2, Qu HJ, Xi RB, **Lu ZJ**<sup>2</sup>, Park PJ & Elledge SJ. (2012) **Systematic Identification of Synergistic Drug Pairs Targeting HIV.** *Nature Biotech* 30:1125-1130. (<sup>2</sup> Lu Lab, School of Life Sciences, Tsinghua University)

  [View Details](http://www.nature.com/nbt/journal/v30/n11/abs/nbt.2391.html)

* Tan X, **Lu ZJ**<sup>2</sup>, Gao G, Xu QK, Hu L<sup>2</sup>, Fellmann C, Li MZ, Qu HJ, Lowe SW, Hannon GJ & Elledge SJ. (2012) **Tiling genomes of pathogenic viruses identifies potent antiviral shRNAs and reveals a role for secondary structure in shRNA efficacy.** *Proc. Natl. Acad. Sci. U. S. A.*, 109(3):869-74. (<sup>2</sup>Lu Lab, School of Life Sciences, Tsinghua University)

  [View Details](http://www.pnas.org/content/109/3/869.abstract)

* The ENCODE Project Consortium# (2012) **An integrated encyclopedia of DNA elements in the human genome.** *Nature* 489:57-74 (# **ZJL** was a member of ENCODE Consortium)

  [View Details](http://www.nature.com/nature/journal/v489/n7414/abs/nature11247.html)
  <img src="./assets/2012.ENCODE_Nature.webp" style="zoom:25%;" />

* He C, Li YX, Zhang G, Gu Z, Yang R, Li J, Lu ZJ, Zhou ZH, Zhang C, Wang J. (2012) **MiRmat: Mature microRNA Sequence Prediction.** *PloS one* 7 (12), e51673.

  [View Details](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0051673)


> —————————————————13————————————————————



## 2010 - 2011 (Postdoc)

* **Lu ZJ**<sup>1</sup>, Yip KY<sup>1</sup>, Wang G, Shou C, Hillier LW, et al. (2011) **Prediction and characterization of noncoding RNAs in C. elegans by integrating conservation, secondary structure, and high-throughput sequencing and array data.** *Genome Research* 21: 276-285. (<sup>1</sup>contributed equally) [modENCODE special issue]

  [View Details](http://genome.cshlp.org/content/21/2/276.full)
  <img src="./assets/2011.ncRNA_GenomeResearch.webp" style="zoom:25%;" />

* Niu W, **Lu ZJ**, Zhong M, Sarov M, Murray JI, et al. (2011) **Diverse transcription factor binding features revealed by genome-wide ChIP-seq in C. elegans.** *Genome Research* 21: 245-254.

  [View Details](http://genome.cshlp.org/content/21/2/245.short)

* The ENCODE Project Consortium (2011) **A User's Guide to the Encyclopedia of DNA Elements (ENCODE).** *PLoS Biology* 9(4):e1001046 (**Zhi J. Lu** is a member of ENCODE)

  [View Details](https://pubmed.ncbi.nlm.nih.gov/21526222/)

* Mu XJ, **Lu ZJ**, Kong Y, Lam HYK, Gerstein MB. (2011) **Analysis of genomic variation in non-coding elements using population-scale sequencing data from the 1000 Genomes Project.** *Nucleic Acids Research* 39 (16):7058-7076.

  [View Details](http://nar.oxfordjournals.org/content/39/16/7058.short)

* Li Y, Du X, **Lu ZJ**, Wu D, Zhao Y, et al. (2011) **Regulatory Feedback Loop of Two phz Gene Clusters through 5' Untranslated Regions in Pseudomonas sp. M18.***PLoS ONE* 6(4): e19413

  [View Details](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0019413)

* Cheng C, Yan KK, Hwang W, Qian J, Bhardwaj N, Joel R, **Lu ZJ**, Niu W, Alves P, Kato M, Gerstein MB. (2011) **Construction and analysis of an integrated regulatory network derived from high-throughput sequencing data.** *PLoS Comput Biol* 7: e1002190

  [View Details](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1002190)

* Gerstein MB<sup>†,</sup>\*, **Lu ZJ<sup>†</sup>**, Van Nostrand EL<sup>†</sup>, Cheng C<sup>†</sup>, Arshinoff BI<sup>†</sup>, et al. (2010) **Integrative Analysis of the Caenorhabditis elegans Genome by the modENCODE Project**  *Science* 330 (6012): 1775-1787  (<sup>†</sup>20 co-first authors, \*15 co-corresponding authors, ~150 authors in total) [Cover Story]

  [View Details](http://www.sciencemag.org/content/330/6012.cover-expansion)
  <img src="./assets/2010.modENCOE_Science.webp" style="zoom:25%;" />

* Zhong M, Niu W, **Lu ZJ**, Sarov M, Murray JI, et al. (2010) **Genome-Wide Identification of Binding Sites Defines Distinct Functions for Caenorhabditis elegans PHA-4/FOXA in Development and Environmental Response.** *PLoS Genetics*  6(2): e1000848.

  [View Details](http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1000848)
  
  

> —————————————————5————————————————————


## 2006 - 2009 (Ph.D.)

<img src="./assets/rnastructure.jpg"  style="zoom:60%;" />

* **Lu ZJ**, Gloor JW, Mathews DH. (2009) **Improved RNA secondary structure prediction by maximizing expedted pair accuracy.** *RNA* 15:1805-1813.

  [View Details](http://rnajournal.cshlp.org/content/15/10/1805.short)

* **Lu ZJ**, Mathews DH. (2008) **Efficient siRNA selection using hybridization thermodynamics.***Nucleic Acids Research* 36:640-647.

  [View Details](http://nar.oxfordjournals.org/content/36/2/640.short)

* **Lu ZJ**, Mathews DH. (2008) **OligoWalk: an online siRNA design tool utilizing hybridization thermodynamics.** *Nucleic Acids Research* 36:W104-W108.

  [View Details](http://nar.oxfordjournals.org/content/36/suppl_2/W104.short)

* **Lu ZJ**, Mathews DH. (2008) **Fundamental differences in the equilibrium considerations for siRNA and antisense oligodeoxynucleotide design.** *Nucleic Acids Research* 36:3738-3745.

  [View Details](http://nar.oxfordjournals.org/content/36/11/3738.short)

* **Lu ZJ**, Turner DH, Mathews DH. (2006) **A set of nearest neighbor parameters for predicting the enthalpy change of RNA secondary structure formation.** *Nucleic Acids Research*34:4912-4924.

  [View Details](http://nar.oxfordjournals.org/content/34/17/4912.short)

  

---

> [More on Google Scholar](http://scholar.google.com/citations?hl=en&user=wrtnAbEAAAAJ)
