---
layout: default
title: Representative Graduates
---

* TOC
{:toc}
---




## 2018' Li, Yang

1. Shuaishuai Teng<sup>1</sup>, **Yang Eric Li<sup>1</sup>**, Ming Yang, Rui Qi, Yiming Huang, Qianyu Wang, Shasha Li, Kequan Lin, Yujing Cheng, Zhi John Lu\*, Dong Wang\*. (2020) Epigenetic Reprogramming of Tissue-Specific Transcription Promotes Metastasis. _**Cell Research**_ (IF=18)  (<sup>1</sup>contributed equally, \*co-corresponding authors)
2. **Yang Eric Li**<sup>1</sup>, Mu Xiao<sup>1</sup>, Binbin Shi<sup>1</sup>, Yu-Cheng T. Yang, Dong Wang, Fei Wang, Marco Marcia, Zhi John Lu*. (2017) Identification of high-confidence RNA
regulatory elements by combinatorial classification of RNA-protein binding sites.
_**Genome Biology**_  18:169. (IF=13) (<sup>1</sup>contributed equally)
3. Yang Yang#, Lei Chen#*, Jin Gu#, Hanshuo Zhang#, Jiapei Yuan, Qiuyu Lian, Guishuai Lv, Siqi Wang, Yang Wu, Yu-Cheng T. Yang, Dongfang Wang, Yang Liu, Jing Tang, Guijuan Luo, **Yang Li**, Long Hu, Xinbao Sun, Dong Wang, Mingzhou Guo, Qiaoran Xi, Jianzhong Xi, Hongyang Wang*, Michael Q. Zhang*, Zhi John Lu* (2017) Recurrently deregulated lncRNAs in hepatocellular carcinoma. **Nature Communications**. *8*, 14421. (SCI, IF=12.124)
4. Yang Wu#, Rihao Qu#, Yiming Huang#, Binbin Shi, Mengrong Liu, **Yang Li**, Zhi John Lu* (2016) RNAex: an RNA secondary structure prediction server enhanced by high-throughput structure probing data. **Nucleic Acids Research**. *44* (W1): W294-W301. (SCI, IF=10.162)
5. Yu-Cheng T. Yang#, Chao Di#, Boqin Hu#, Meifeng Zhou, Yifang Liu, Nanxi Song, **Yang Li**, Jumpei Umetsu, Zhi John Lu*. (2015) CLIPdb: a CLIP-seq database for protein-RNA interactions. **BMC Genomics**. 16 (1):51-58. (SCI, IF=3.729)
6. Long Hu, Chao Di, Mingxuan Kai, Yu-Cheng T. Yang, **Yang Li**, Yunjiang Qiu, Xihao Hu, Kevin Y. Yip, Michael Q. Zhang, Zhi John Lu. (2015) A common set of distinct features that characterize noncoding RNAs across multiple species. **Nucleic Acids Research**. *43* (1): 104-114. (SCI, IF=10.162)



## 2017' Yang, Yang

1. **Yang Y**, Yang YCT, Yuan J, Lu ZJ and Li JJ (2017) Large-scale mapping of mammalian transcriptomes identifies conserved genes associated with different cell differentiation states. **Nucleic Acids Research** 45 (4): 1657-1672
2. **Yang Y**,ChenL,GuJ,ZhangH,YuanJ,LianQ,LvG,WangS,WuY,YangYCT, Wang D, Liu Y, Tang J, Luo G, Li Y, Hu L, Sun X, Wang D, Guo M, Xi Q, Xi J, Wang H, Zhang MQ and Lu ZJ (2017) Recurrently deregulated lncRNAs in hepatocellular carcinoma**.** **Nature Communications** 8, 14421
3. CaoB,SongN,ZhangM,DiC,**Yang Y**,LuYou,ChenR,LuZJ,GuoM(2016) Systematic study of novel lncRNAs in different gastrointestinal cancer cells. **Discovery Medicine,** 21(115): 159-71
4. TangZ,WuY,YangYL,YangYC,WangZ,YuanJ,**Yang Y**,HuaC,FanX,NiuG, Zhang Y, Lu ZJ, Li K (2017) Comprehensive analysis of long non-coding RNAs highlights their spatio-temporal expression patterns and evolutional conservation in Sus Scrofa. **Scientific Reports,** 7: 43166
5. Xi X, Li T, Huang Y, Sun J, Zhu Y, **Yang Y** and Lu ZJ (2017) RNA biomarkers: Frontiers of Precision Medicine for Cancer. **Non-Coding RNA** 3,9

## 2016' Hu, Long

1. Zhiyu Xu<sup>1</sup>, **Long Hu<sup>1</sup>**, Binbin Shi, SiSi Geng, Longchen Xu, Dong Wang\* and Zhi John Lu\* (2018) Ribosome elongating footprints denoised by wavelet transform comprehensively characterize dynamic cellular translation events. ***Nucleic Acids Research*** 46 (18), e109-e109  (<sup>1</sup>contributed equally, \*co-corresponding authors)
2. **Long Hu**, Zhiyu Xu, Boqin Hu and Zhi John Lu (2017) COME: a robust coding potential calculator for lncRNA identification and characterization based on multiple features. ***Nucleic Acids Research*** 45 (1): e2
3. **Hu L**,Di C,Kai MX,Yang T.YC,Li Y,Qiu YJ,Hu XH,Yip K,Zhang M. QW&Lu ZJ. (2014) A common set of distinct features that characterize noncoding RNAs across multiple species. **Nucleic Acids Research** doi: 10.1093/nar/gku1316
4. Tan X,**Hu L**,LuquetteLJ,GaoG,LiuYF,QuHJ,XiRB,LuZJ,ParkPJ&ElledgeSJ. (2012) Systematic Identification of Synergistic Drug Pairs Targeting HIV. **Nature Biotech,** 30:1125-1130
5. Tan X,Lu Z,Gao G,Xu QK,**Hu L**,FellmannC,LiMZ,QuHJ,LoweSW,HannonGJ& Elledge SJ. (2012) Tiling genomes of pathogenic viruses identifies potent antiviral shRNAs and reveals a role for secondary structure in shRNA efficacy. **Proc. Natl. Acad. Sci. U. S. A.**, 109(3): 869-74
6. Gerstein MB, ... , **Hu L**, ... (2014) Comparative analysis of the transcriptome across distant speices. **Nature**, 512:445-448
7. DiC,YuanJP,WuY,LiJR,LinHX,**Hu L**,ZhangT,QiYJ,GersteinMB,GuoY,LuZ. (2014) Characterization of stress-responsive lncRNA in Arabidopsis Thaliana by integrating expression, epigenetic and structural features. **The Plant Journal**, 80(5): 848-61

## 2016' Yang, YC

1.	Yumin Zhu<sup>1</sup>, Gang Xu<sup>1</sup>, **Yucheng T Yang<sup>1</sup>**, Zhiyu Xu, Xinduo Chen, Binbin Shi, Daoxin Xie, **Zhi John Lu\***, Pengyuan Wang\* (2019) POSTAR2: deciphering the post-transcriptional regulatory logics ***Nucleic Acids Research*** (D1):D203-D211  (<sup>1</sup>contributed equally, \*co-corresponding authors)
2.	Boqin Hu<sup>1</sup>, **Yu-Cheng T. Yang<sup>1</sup>**, Yiming Huang, Yumin Zhu and Zhi John Lu (2017) POSTAR: a platform of post-transcriptional regulation coordinated by RNA binding proteins. ***Nucleic Acids Research*** 45 (D1): D104-D114 (<sup>1</sup>contributed equally)
3.	**Yang YC**, Di C, Hu B, Zhou M, Liu Y, Song N, Li Y, Umetsu J and Lu ZJ (2015) CLIPdb: a CLIP-seq database for protein-RNA interactions. ***BMC Genomics.***
4.	**Yang YC**, Umetsu J and Lu ZJ (2014) Global signatures of protein binding on structured RNAs in Saccharomyces cerevisiae. ***Science China Life Sciences***
5.	Hu L, Di C, Kai M, **Yang YC**, Li Y, Qiu Y, Hu X, Yip KY, Zhang MQ and Lu ZJ (2015) A common set of distinct features that characterize noncoding RNAs across multiple species. _**Nucleic Acids Research**_
6.	Freeberg MA, Han T, Moresco JJ, Kong A, **Yang YC**, Lu ZJ, Yates JR and Kim JK. (2013) Pervasive and dynamic protein binding sites of the mRNA transcriptome in Saccharomyces cerevisiae. _**Genome Biology**_
