<!DOCTYPE html>
<html>
    <head>
        <title>Software @ Lu lab</title>

        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta content="viewport" content="width=device-width, initial-scale=1">

        <!-- Font.css -->
        <link rel="stylesheet" href="css/font.css">
        <!-- Animate.css -->
        <link rel="stylesheet" href="css/animate.css">
        <!-- Icomoon Icon Fonts-->
        <link rel="stylesheet" href="css/icomoon.css">
        <!-- Bootstrap  -->
        <link rel="stylesheet" href="css/bootstrap.css">
        <!-- Theme style  -->
        <link rel="stylesheet" href="css/style.css">
        <!-- Modernizr JS -->
        <script src="js/modernizr-2.6.2.min.js"></script>
        <!-- FOR IE9 below -->
        <!--[if lt IE 9]
        <script src="js/respond.min.js"></script>
        [endif]
        -->

        <link rel="stylesheet" href="css/software.css">
    </head>

    <body>

        <div id="page">
            <div id="fh5co-aside" style="background-image: url(images/dna-high.jpg)">
                <div class="overlay">
                    <br>
                    <snap style="font-size:20px; padding-left: 80px; "><b>Lu Lab</b></snap></div>
                <nav role="navigation">
                    <ul>
                        <li><a class="foot al" href="http://bioinfo.life.tsinghua.edu.cn"><span class="icon-home"></span></a></li>
                    </ul>
                </nav>
                <div class="featured">
                    <h1>Welcome to <a href="http://bioinfo.life.tsinghua.edu.cn"><b>Lu Lab</b></a> software page!</h1>
                    <!-- <h2><a href="http://life.tsinghua.edu.cn/english/faculty/faculty/1599.html">Leader: Zhi John Lu</a></h2> -->
                    <span><b>Lu Lab</b> is interested in developing bioinformatics technologies and practicing evidence-based medicine. </span>
                    <span>Here, we develop <b>bioinformatic softwares</b> about biological system, and applying them to RNA biology and Precision Medicine. </span>
                </div>
            </div>
            
            <div id="fh5co-main-content">
                <div class="fh5co-post">

                    <div class="fh5co-entry" style="padding-left: 45px; padding-top: 15px; padding-bottom: 70px; ">
                        <div>
                            <!-- <div > -->
                            <p class="btn-app-stroe row" style="position:fixed; width: 88%;">
                                <a class="btn btn-sm btn-success col-xs-4 col-sm-4 col-md-4 col-lg-2" style="font-size: 18px; opacity:0.8;"  href="#RNAfinder" role="button">RNAfinder</a>
                                <a class="btn btn-sm btn-success col-xs-4 col-sm-4 col-md-4 col-lg-2" style="font-size: 18px; opacity:0.8;" href="#RNAtarget" role="button" >RNAtarget</a>
                                <a class="btn btn-sm btn-success col-xs-4 col-sm-4 col-md-4 col-lg-2" style="font-size: 18px; opacity:0.8;" href="#RNAstructurome" role="button" >RNAstructurome</a>
                                <a class="btn btn-sm btn-success col-xs-4 col-sm-4 col-md-4 col-lg-2" style="font-size: 18px; opacity:0.8;" href="#RNAmed" role="button">RNAmed</a>
                            </p>
                        </div>
                    </div>

                    <!-- RNAfinder -->
                    <div class="fh5co-entry padding"  id="RNAfinder" style="padding-top: 100px;">
                        <div style="font-size: 45px;">
                            <b>RNAfinder</b>
                        </div>
                        <div >
                            <div style="font-size: 30px;">
                                <a href="https://lulab.github.io/Ribowave"><b>Ribowave</b></a>
                            </div>
                            <span class="fh5co-post-date">Zhiyu Xu, Long Hu, Binbin Shi, SiSi Geng, et al. Nucleic Acids Research (2018)</span>
                            <p>RiboWave analyses Ribosome profiling data (Ribo-seq). It utilizes wavelet transform to denoise the original signal by extracting 3-nt periodicity of ribosomes (i.e. signal frequency) and precisely locate their footprint.</p> 
                        </div>
                        <div>
                            <div style="font-size: 30px;">
                                <a href="https://github.com/lulab/COME"><b>COME</b></a>
                            </div>
                            <span class="fh5co-post-date">Long Hu, et al. Nucleic Acids Research (2017)</span>
                            <p>COME is a tool to calculate COding potential from Multiple fEatures for a given transcript. The models in COME were trained on mRNAs and long ncRNAs (lncRNAs). </p>
                        </div>
                        <div>
                            <div style="font-size: 30px;">
                                <a href="https://www.ncrnalab.org/RNAfinder/"><b>RNAfeature</b></a>
                            </div>
                            <span class="fh5co-post-date">Long Hu, et al. Nucleic Acids Research (2015)</span>
                            <p>RNAfeature provides a common set of conserved features for ncRNAs across multiple species. The models in RNAfeature were trained on canonical ncRNAs (e.g.,tRNAs,rRNAs, miRNAs, snRNAs, snoRNAs, 7SK RNAs, Y RNAs).</p>
                        </div>
                    </div>
                    
                    <!--RNAtarget-->
                    <div class="fh5co-entry padding" id="RNAtarget" style="padding-top: 100px;"> 
                        <div style="font-size: 45px;">
                            <b >RNAtarget</b>
                        </div>
                        <div>
                            <div style="font-size: 30px;">
                                <a href="https://lulab.github.io/RBPgroup"><b>RBPgroup</b></a>
                            </div>
                            <span class="fh5co-post-date">Yang Eric Li, Mu Xiao, Binbin Shi, Yu-Cheng T. Yang, et al. Genome Biology (2017)</span>
                            <p>RBPgroup is a soft-clustering method on various CLIP-seq datasets, in order to group together RBPs that specifically bind the same RNA sites. Our approach links proteins and RNA motifs known to possess similar biochemical and cellular properties and can, when used in conjunction with additional experimental data, identify high- confidence RBP groups and their associated RNA regulatory elements.</p>
                        </div>
                        <div>
                            <div style="font-size: 30px;">
                                <a href="http://lulab.life.tsinghua.edu.cn/postar/"><b>POSTAR</b></a>
                            </div>
                            <span class="fh5co-post-date">Boqin Hu, Yu-Cheng T. Yang, et al. Nucleic Acids Research (2017)</span>
                            <p>POSTAR (CLIPdb 2) is a platform for exploring post-transcriptional regulation coordinated by RNA-binding proteins. It enables the experimental biologists to connect protein-RNA interactions with multi-layer information of post-transcriptional regulation and functional genes, and helps them generate novel hypotheses about the post-regulatory mechanisms of phenotypes and diseases.</p>
                        </div>
                        <div>
                            <div style="font-size: 30px;">
                                <a href="http://lulab.life.tsinghua.edu.cn/postar/rbp2.php"><b>CLIPdb</b></a>
                            </div>
                            <span class="fh5co-post-date">Yu-Cheng T. Yang, Chao Di, Boqin Hu, et al. BMC Genomics (2015)</span>
                            <p>CLIPdb is an integrative resource of CLIP-seq studies. It aims to characterize the regulatory networks between RNA binding proteins (RBPs) and various RNA transcript classes by integrating large amounts of CLIP-seq (including HITS-CLIP, PAR-CLIP and iCLIP as variantions) data sets. </p>
                        </div>
                    </div>
                    
                    <!--RNAstructurome-->
                    <div class="fh5co-entry padding" id="RNAstructurome" style="padding-top: 100px;">
                        <div style="font-size: 45px;">
                            <b >RNAstructurome</b>
                        </div>
                        <div>
                            <div style="font-size: 30px;">
                                <a href="#"><b>RNAex</b></a>
                            </div>
                            <span class="fh5co-post-date">Yang Wu, Rihao Qu, Yiming Huang, â€‹et al. Nucleic Acids Research (2016)</span>
                            <p>under construction.<br />RNAex is an RNA secondary structure prediction server enhanced by high-throughput experimental data. We have re-mapped raw data of the published probing experiments to the whole genome, thus users can predict secondary structures for novel RNA transcripts.</p>
                        </div>
                        <div>
                            <div style="font-size: 30px;">
                                <a href="https://github.com/lulab/RME"><b>RME</b></a>
                            </div>
                            <span class="fh5co-post-date">Yang Wu, Binbin Shi, et al. Nucleic Acids Research (2015)</span>
                            <p>RME is a tool for RNA secondary structure prediction with multiple types of experimental probing data. It is based on the <a href="http://rna.urmc.rochester.edu/RNAstructure.html">RNAstructure</a> package. It also provides preprocessing scripts for transforming the SHAPE, PARS and DMS-seq data into pairing probability according a posterior probabilistic model. Moreover, it also contains a utility for optimizing the parameters of RME by RME-Optimize.</p>
                        </div>
                    </div>

                    <!--RNAmed-->
                    <div class="fh5co-entry padding" id="RNAmed" style="padding-top: 100px;">
                        <div style="font-size: 45px;">
                            <b >RNAmed</b>
                        </div>
                        <div>
                            <div style="font-size: 30px;">
                                <a href="https://github.com/ShangZhang/exVariance"><b>exVariance</b></a>
                            </div>
                            <span class="fh5co-post-date">Shang Zhang, et al.</span>
                            <p>under construction</p>
                        </div>
                        <div>
                            <div style="font-size: 30px;">
                                <a href="http://lulab.life.tsinghua.edu.cn/exrpanel"><b>exRpanel</b></a>
                            </div>
                            <span class="fh5co-post-date">Gang Xu, Shang Zhang, Hongke Wang, et al. (2019)</span>
                            <p>exRpanel is developed to discover biomarker panels that can be used for disease diagnosis based on exRNA expression data.</p>
                        </div>
                        <div>
                            <div style="font-size: 30px;">
                                <a href="https://lulab.github.io/exSEEK"><b>exSEEK</b></a>
                            </div>
                            <span class="fh5co-post-date">Binbin Shi, Jingyi Cao, Xupeng Chen, et al. (2019)</span>
                            <p>exSEEK is an integrative computational framework for identifying extracellular RNA biomarkers in liquid biopsy.</p>
                        </div>
                        <div>
                            <div style="font-size: 30px;">
                                <a href="http://lulab.life.tsinghua.edu.cn/ccg/"><b>CCG</b></a>
                            </div>
                            <span class="fh5co-post-date">(2016)</span>
                            <p>CCG is an assembly resource of coding and noncoding genes associated with cancer.</p>
                        </div>
                    </div>
                </div>
            </div>
        </div>

        <div class="gototop js-top">
            <a href="#" class="js-gotop"><i class="icon-arrow-up"></i></a>
        </div>

        <!-- jQuery -->
        <script src="js/jquery.min.js"></script>
        <!-- jQuery Easing -->
        <script src="js/jquery.easing.1.3.js"></script>
        <!-- Bootstrap -->
        <script src="js/bootstrap.min.js"></script>
        <!-- Waypoints -->
        <script src="js/jquery.waypoints.min.js"></script>
        <!-- Stellar Parallax -->
        <script src="js/jquery.stellar.min.js"></script>
        <!-- Main -->
        <script src="js/main.js"></script>


        <style type="text/css">
            .btn-app-stroe a{
                display: inline;
                border-radius: 10px;
                font-weight: bold;
                margin-left: 20px;
                font-size: 20px;
            }
            .btn-success {
                color: #fff;
                background-color: #b4b4b4;
                border-color: #8d8d8d;
            }
            
        </style>
    </body>
</html>