---
layout: default
title:  Representative Publications
---

**Table of Contents:**

* TOC
{:toc}


<div align="middle">
<img src="../projects/projects.webp" style="zoom:20%;" />
</div>


> [Full list of publications](./index.md)
>
> [Software](../software/index.md)
>
> [代表性文章中文解读](../news/#news)

---




## A. [RNA-Finder] Bioinfo-Driven Precision Medicine


### A1. (Cell-free) RNA Biomarker

- Pengfei Bao<sup>1</sup>, Taiwei Wang<sup>1</sup>, Xiaofan Liu<sup>1</sup>, Shaozhen Xing, Hanjin Ruan, Hongli Ma, Yuhuan Tao, Qing Zhan, Efres Belmonte-Reche, Lizheng Qin, Zhengxue Han, Minghui Mao\*, Mengtao Li\* & **Zhi John Lu**\* (2025) Peak analysis of cell-free RNA finds recurrently protected narrow regions with clinical potential. **Genome Biology** 26, 119 (<sup>1</sup>contributed equally, \*co-corresponding authors)  Github: https://github.com/lulab/cfPeak
- Hongke Wang<sup>1</sup>, Qing Zhan<sup>1</sup>, Meng Ning<sup>1</sup>, Hongjie Guo<sup>1</sup>, Qian Wang, Jiuliang Zhao, Pengfei Bao, Shaozhen Xing, Shanwen Chen, Shuai Zuo, Xuefeng Xia, Mengtao Li\*, Pengyuan Wang\*, **Zhi John Lu**\* (2024) Depletion-assisted multiplexed cell-free RNA sequencing reveals distinct human and microbial signatures in plasma versus extracellular vesicles **Clinical and Translational Medicine** 14:e1760 (<sup>1</sup>contributed equally, \*co-corresponding authors) 
- Xiaofan Liu<sup>1</sup>, Yuhuan Tao<sup>1</sup>, Zilin Cai, Pengfei Bao, Hongli Ma, Kexing Li, Mengtao Li\*, Yunping Zhu\*, **Zhi John Lu**\*  (2024) Pathformer: a biological pathway informed Transformer for disease diagnosis and prognosis using multi-omics data **Bioinformatics** 40(5), btae316 (<sup>1</sup>contributed equally, \*co-corresponding authors) Github: https://github.com/lulab/Pathformer
- Mingyang Li<sup>1</sup>, Tianxiu Zhou<sup>1</sup>, Mingfei Han<sup>1</sup>, Hongke Wang, Pengfei Bao, Yuhuan Tao, Xiaoqing Chen, Guansheng Wu, Tianyou Liu, Xiaojuan Wang, Qian Lu\*, Yunping Zhu\*, **Zhi John Lu**\* (2024) cfOmics: a cell-free multi-Omics database for diseases.  **Nucleic Acids Research** 52(D1):D607-D621 (<sup>1</sup>contributed equally, \*co-corresponding authors)  Website: http://cfOmics.ncRNAlab.org
- Yuhuan Tao<sup>1</sup>, Shaozhen Xing<sup>1</sup>, Shuai Zuo<sup>1</sup>, Pengfei Bao, Yunfan Jin, Yu Li, Yingchao Wu, Shanwen Chen, Xiaojuan Wang, Yumin Zhu, Ying Feng, Xiaohua Zhang, Xianbo Wang, Qiaoran Xi, Qian Lu\*, Pengyuan Wang\*, **Zhi John Lu**\*  (2023) Cell-free multi-omics analysis reveals potential biomarkers in gastrointestinal cancer patients’ blood. **Cell Reports Medicine** 4(11):101281 (<sup>1</sup>contributed equally, \*co-corresponding authors) 
- Xing, S.<sup>1</sup>, Zhu, Y.<sup>1</sup>, You, Y., Wang, S., Wang, H., Ning, M., Jin, H., Liu, Z., Zhang, X., Yu, C. \* & **Lu, Z. J.**\* (2023). Cell-free RNA for the liquid biopsy of gastrointestinal cancer. **WIREs RNA** 14(5):e1791 (<sup>1</sup>contributed equally, \*co-corresponding authors) 
- Chun Ning<sup>1</sup>, Peng Cai<sup>1</sup>, Xiaofan Liu<sup>1</sup>, Guangtao Li<sup>1</sup>, Pengfei Bao, Lu Yan, Meng Ning, Kaichen Tang, Yi Luo, Hua Guo, Lu Chen\*, **Zhi John Lu** \*, Jianhua Yin\* (2023) A Comprehensive Evaluation of Full-spectrum Cell-free RNAs highlights cell-free RNA fragments for Early-Stage Hepatocellular Carcinoma Detection **eBioMedicine** 93:104645 (<sup>1</sup>contributed equally, \*co-corresponding authors) 
- Shanwen Chen<sup>1</sup>, Yunfan Jin<sup>1</sup>, Siqi Wang<sup>1</sup>, Shaozhen Xing<sup>1</sup>, …, Zhenjiang Zech Xu\*, **Zhi John Lu\*** and Pengyuan Wang\* (2022) Cancer Type Classification Using Plasma Cell Free RNAs Derived from Human and Microbes. **eLife** 11:e75181 (<sup>1</sup>contributed equally, \*co-corresponding authors) (IF: 8.7)
- Yumin Zhu<sup>1</sup>, Siqi Wang<sup>1</sup>, Xiaochen Xi<sup>1</sup>, Minfeng Zhang<sup>1</sup>, Xiaofan Liu, Weina Tang, Peng Cai, Shaozhen Xing, Pengfei Bao, Yunfan Jin, Weihao Zhao, Yinghui Chen, Huanan Zhao, Xiaodong Jia, Shanshan Lu, Yinying Lu, Lei Chen\*, Jianhua Yin\*, **Zhi John Lu**\* (2021) Integrative analysis of long extracellular RNAs reveals a detection panel of noncoding RNAs for liver cancer. **Theranostics** 11(1):181-193 (<sup>1</sup>contributed equally, \*co-corresponding authors) (IF: 11.556) (Patent Protected)
- Shuaishuai Teng<sup>1</sup>, Yang Eric Li<sup>1</sup>, …, **Zhi John Lu\*** and Dong Wang\* (2020) Tissue-specific transcription reprogramming promotes liver metastasis of colorectal cancer. **Cell Research** 30(1):34-49 (<sup>1</sup>contributed equally, *co-corresponding authors) (IF: 25.6)
- Chang Tan<sup>1</sup>, Jingyi Cao<sup>1</sup>, Lu Chen<sup>1</sup>, Xiaochen Xi, Siqi Wang, Yumin Zhu, Liuqing Yang, Longteng Ma, Dong Wang, Jianhua Yin\*, Ti Zhang\* and **Zhi John Lu\*** (2019) Noncoding RNAs serve as diagnosis and prognosis biomarkers for hepatocellular carcinoma. **Clinical Chemistry** 65(7):905-915  (<sup>1</sup>contributed equally, \*co-corresponding authors) (IF: 8.3) (Patent Protected)
- Yang Yang<sup>1</sup>, Lei Chen<sup>1</sup>, Jin Gu<sup>1</sup>, Hanshuo Zhang<sup>1</sup>, …, Hongyang Wang\*, Michael Q. Zhang\* and **Zhi John Lu\*** (2017) Recurrently deregulated lncRNAs in hepatocellular carcinoma.**Nature Communications** 8, 14421 (<sup>1</sup>contributed equally, \*co-corresponding authors) (IF: 17.69)


### A2. ncRNA Method

- Yu-Xiang Chen1, Zhi-yu Xu1, Xueliang Ge, Suparna Sanyal, **Zhi John Lu\*** and Babak Javid* (2020) Selective translation by alternative bacterial ribosomes. **PNAS** 117(32):19487-19496 (1contributed equally, *co-corresponding authors) (IF: 12.78)
- \[**Algorithm V3**\] Zhiyu Xu1, Long Hu1, Binbin Shi, SiSi Geng, Longchen Xu, Dong Wang* & **Zhi John Lu\*** (2018) Ribosome elongating footprints denoised by wavelet transform comprehensively characterize dynamic cellular translation events. **Nucleic Acids Research** 46 (18), e109-e109 (IF: 19.16) Github: https://github.com/lulab/Ribowave
- Yang Yang1, Lei Chen1, Jin Gu1, Hanshuo Zhang1, …, Hongyang Wang*, Michael Q. Zhang* and **Zhi John Lu\*** (2017) Recurrently deregulated lncRNAs in hepatocellular carcinoma.**Nature Communications** 8, 14421 (1contributed equally) (IF: 17.69)
- \[**Algorithm V2**\]  Long Hu, Zhiyu Xu, Boqin Hu and **Zhi John Lu\*** (2017) COME: a robust coding potential calculation tool for lncRNA identification and characterization based on multiple features. **Nucleic Acids Research** 45(1): e2 (IF: 19.16) Github: https://github.com/lulab/COME
- \[**Algorithm V1**\]  Long Hu, Chao Di, Mingxuan Kai, Yu-Cheng T. Yang, Yang Li, Yunjiang Qiu, Xihao Hu, Kevin Y. Yip, Michael Q. Zhang and **Zhi John Lu\*** (2015) A common set of distinct features that characterize noncoding RNAs across multiple species. ***Nucleic Acids Research\*** 43(1): 104-114. (IF: 19.16) Github: https://github.com/lulab/RNAfeature
- Gerstein MB, Joel Rozowsky, Koon-Kiu Yan, …, **Lu Z,** …, Robert Waterston (2014) Comparative Analysis of the Transcriptome across Distant Species. **Nature** 512(7515):445-448.
- ENCODE Project Consortium**#**. (2012) An Integrated Encyclopedia of DNA Elements in the Human Genome. **Nature** 489, 57–74. (**#Lu Z** was a member of ENCODE Consortium) [Cover Story]
- **[AlgorithmV0]** **Lu ZJ**<sup>1</sup>, Yip KY<sup>1</sup>, Wang G, Shou C, Hillier LW, et al. (2011) Prediction and characterization of noncoding RNA in *C. elegans* by integrating conservation, secondary structure and high-throughput sequencing and array data. **Genome Research** 21: 276-2857 (<sup>1</sup>contributed equally) (IF: 10.98)
- **[Algorithm V0]** Gerstein MB<sup>†,</sup>\*, **Lu ZJ<sup>†</sup>**, Van Nostrand EL<sup>†</sup>, Cheng C<sup>†</sup>, Arshinoff BI<sup>†</sup>, et al. (2010) Integrative Analysis of the Caenorhabditis elegans Genome by the modENCODE Project.  **Science** 330 (6012): 1775-1787  (<sup>†</sup>20 co-first authors, \*15 co-corresponding authors, ~150 authors in total) [Cover Story]




## B. [RNA-Talk] AI-Driven RNA Modeling & Drug Design

### B1. RNA Interaction (trans-pair) and Regulation Network 

#### RNA-RNA (trans-pair)

- [**Language Model**] Yilan Bai, Haochen Zhong, Taiwei Wang, **Zhi John Lu** (2024) OligoFormer: An accurate and robust prediction method for siRNA design. **Bioinformatics** 40 (10), btae577. software: [*OligoFormer*](https://github.com/lulab/OligoFormer)
- Tan X, **Lu ZJ <sup>2</sup>**, Gao G, Xu QK, Hu L2, Fellmann C, Li MZ, Qu HJ, Lowe SW, Hannon GJ & Elledge SJ. (2012) Tiling genomes of pathogenic viruses identifies potent antiviral shRNAs and reveals a role for secondary structure in shRNA efficacy. **PNAS**, 109(3): 869-74 (<sup>2</sup>Lu Lab, School of Life Sciences, Tsinghua University) (IF: 12.78)
- \[**Server**\] **Lu ZJ** and Mathews DH (2008)  OligoWalk: an online siRNA design tool utilizing hybridization thermodynamics.   **Nucleic Acids Research**  36:W104-W108. (IF: 19.16) website: [*OligoWalk*](https://rna.urmc.rochester.edu/cgi-bin/server_exe/oligowalk/oligowalk_form.cgi)
- **\[Algorithm\] Lu ZJ** and Mathews DH (2008) Efficient siRNA selection using hybridization thermodynamics. **Nucleic Acids Research** 36:640-647. (IF: 19.16) software: [*RNAstructure*](https://rna.urmc.rochester.edu/RNAstructure.html)
- **[Biophysical Model]** **Lu ZJ** and Mathews DH (2008) Fundamental differences in the equilibrium considerations for siRNA and antisense oligodeoxynucleotide design. **Nucleic Acids Research** 36:3738-3745. (IF: 19.16)


#### RNA-Protein (POSTAR/CLIPdb)

- **[Database V3]** Weihao Zhao<sup>1</sup>, Shang Zhang<sup>1</sup>, …, Yucheng T Yang*, **Zhi John Lu\*** (2022) POSTAR3: an updated platform for exploring post-transcriptional regulation coordinated by RNA-binding proteins. **Nucleic Acids Research** 50(D1):D287-D294 (IF: 19.16) Website: http://POSTAR3.ncRNAlab.org
- Rui Xiao<sup>1</sup>, Jia-Yu Chen<sup>1</sup>, Zhengyu Liang<sup>1</sup>, Daji Luo, Geng Chen, **Zhi John Lu**, …, Michael Q. Zhang, Yu Zhou and Xiang-Dong Fu (2019) Pervasive Chromatin-RNA Binding Protein Interactions Enable RNA-Based Regulation of Transcription. **Cell** 178, 107–121
- **[Database V2]** Yumin Zhu1, Gang Xu1, Yucheng T Yang1, Zhiyu Xu, Xinduo Chen, Binbin Shi, Daoxin Xie, **Zhi John Lu\*** and Pengyuan Wang* (2019) POSTAR2: deciphering the post-transcriptional regulatory logics. **Nucleic Acids Research** (D1):D203-D211 (1contributed equally) (IF: 19.16) Website: http://POSTAR2.ncRNAlab.org
- **[Mathematical Model]** Yang Eric Li<sup>1</sup>, Mu Xiao<sup>1</sup>, Binbin Shi<sup>1</sup>, Yu-Cheng T. Yang, Dong Wang, Fei Wang, Marco Marcia and **Zhi John Lu*** (2017) Identification of high-confidence RNA regulatory elements by combinatorial classification of RNA-protein binding sites. **Genome Biology** 18:169 (1contributed equally) (IF: 17.91) Github: https://github.com/lulab/RBPgroup
- **[Database V1]** Boqin Hu<sup>1</sup>, Yu-Cheng T. Yang<sup>1</sup>, Yiming Huang, Yumin Zhu and **Zhi John Lu\*** (2017) POSTAR: a platform of post-transcriptional regulation coordinated by RNA binding proteins. **Nucleic Acids Research** 45 (D1): D104-D114 (1contributed equally) (IF: 19.16) Website: http://POSTAR.ncRNAlab.org


### B2. RNA Structure (cis-pair) Prediction & Enzyme Design

- **[Server]** Yang Wu<sup>1</sup>, Rihao Qu<sup>1</sup>, Yiming Huang<sup>1</sup>, Binbin Shi, Mengrong Liu, Yang Li and **Zhi John Lu\*** (2016) RNAex: an RNA secondary structure prediction server enhanced by high-throughput structure probing data. **Nucleic Acids Research** 44 (W1): W294-W301 (<sup>1</sup>contributed equally) (IF: 19.16)
- **[Algorithm]** Yang Wu<sup>1</sup>, Binbin Shi<sup>1</sup>, Xinqiang Ding, Tong Liu, Xihao Hu, Kevin Y. Yip, Zheng Rong Yang, David H. Mathews and **Zhi John Lu\*** (2015) Improved prediction of RNA secondary structure by integrating the free energy model with restraints derived from experimental probing data. **Nucleic Acids Research** 43(15): 7247-59. (<sup>1</sup>contributed equally) (IF: 19.16) Github: https://github.com/lulab/RME
- **\[Algorithm\]** **Lu ZJ**, Gloor JW and Mathews DH (2009) Improved RNA secondary structure prediction by maximizing expected pair accuracy. **RNA** 15: 1805-1813. (IF: 4.9) software: [*RNAstructure*](https://rna.urmc.rochester.edu/RNAstructure.html)
- **[Biophysical Model]** **Lu ZJ**, Turner DH and Mathews DH (2006) A set of nearest neighbor parameters for predicting the enthalpy change of RNA secondary structure formation. **Nucleic Acids Research** 34:4912-4924. (IF: 19.16)


### B3. Drug Design Targeting RNA

#### Small RNA drug targeting RNA

- [**Language Model**] Yilan Bai, Haochen Zhong, Taiwei Wang, **Zhi John Lu** (2024) OligoFormer: An accurate and robust prediction method for siRNA design. **Bioinformatics** 40 (10), btae577. software: [*OligoFormer*](https://github.com/lulab/OligoFormer)
- Tan X, **Lu ZJ <sup>2</sup>**, Gao G, Xu QK, Hu L2, Fellmann C, Li MZ, Qu HJ, Lowe SW, Hannon GJ & Elledge SJ. (2012) Tiling genomes of pathogenic viruses identifies potent antiviral shRNAs and reveals a role for secondary structure in shRNA efficacy. **PNAS**, 109(3): 869-74 (<sup>2</sup>Lu Lab, School of Life Sciences, Tsinghua University) (IF: 12.78)
- \[**Server**\] **Lu ZJ** and Mathews DH (2008)  OligoWalk: an online siRNA design tool utilizing hybridization thermodynamics.   **Nucleic Acids Research**  36:W104-W108. (IF: 19.16) website: [*OligoWalk*](https://rna.urmc.rochester.edu/cgi-bin/server_exe/oligowalk/oligowalk_form.cgi)
- **\[Algorithm\] Lu ZJ** and Mathews DH (2008) Efficient siRNA selection using hybridization thermodynamics. **Nucleic Acids Research** 36:640-647. (IF: 19.16) software: [*RNAstructure*](https://rna.urmc.rochester.edu/RNAstructure.html)
- **[Biophysical Model]** **Lu ZJ** and Mathews DH (2008) Fundamental differences in the equilibrium considerations for siRNA and antisense oligodeoxynucleotide design. **Nucleic Acids Research** 36:3738-3745. (IF: 19.16)



#### Small molecule drug targeting RNA

- Hongli Ma, Letian Gao, Yunfan Jin, Jianwei Ma, Yilan Bai, Xiaofan Liu, Pengfei Bao, Ke Liu, Zhenjiang Zech Xu\* and **Zhi John Lu**\*. (2025) RNA–ligand interaction scoring via data perturbation and augmentation modeling. **Nature Computational Science** 10.1038/s43588-025-00820-x (\*co-corresponding authors) 
- Kequan Lin<sup>1</sup>, Lu Li<sup>1</sup>, Yifei Dai, Huili Wang, Shuaishuai Teng, Xilinqiqige Bao, **Zhi John Lu**\* and Dong Wang\* (2019) A comprehensive evaluation of connectivity methods for L1000 data. **Briefings in Bioinformatics** bbz129 (<sup>1</sup>contributed equally) (IF: 11.62)
- Tan X, Hu L<sup>2</sup>, Luquette LJ, Gao G, Liu YF2, Qu HJ, Xi RB, **Lu ZJ**<sup>2</sup>, Park PJ & Elledge SJ. (2012) Systematic Identification of Synergistic Drug Pairs Targeting HIV. **Nature Biotech.** 30:1125-1130 (<sup>2</sup>Lu Lab, School of Life Sciences, Tsinghua University) (IF: 54.36)

